Program.cs

```diff
@@ -377,7 +377,20 @@ static void Setup(string username = "")
 						Console.WriteLine("error: Unable to add your phone number. Steam returned GeneralFailure");
 						return;
 					case AuthenticatorLinker.LinkResult.AuthenticatorPresent:
-						Console.WriteLine("error: Can't link authenticator, remove the previous authenticator.");
+						Console.WriteLine("An authenticator is already present.");
+						Console.WriteLine("If you would like to remove the current authenticator, enter your revocation code (Rxxxxx).");
```

Comment:
Sorry, could you explain why? There is no meaningful way we can proceed without deactivating the current authenticator.